mne_bids/read.py

```diff
@@ -95,9 +95,16 @@ def _handle_events_reading(events_fname, raw):
     else:
         descriptions = 'n/a'
 
+    # Check for 'n/a' strings before converting to float
+    ons = list(map(lambda on: np.nan if on == 'n/a' else on,
+               events_dict['onset']))
+
+    durs = list(map(lambda dur: np.nan if dur == 'n/a' else dur,
+                events_dict['duration']))
```

Comment:
should this not be fixed in _read_tsv?

Reply:
yes true. Good catch! I'll do it tomorrow

Reply:
actually, it might be better to fix it here. Because looking at the tsv_handler module, I found that we currently read and write `'n/a'` and also represent NANs internally with `'n/a'`, see e.g. https://github.com/mne-tools/mne-bids/blob/2ca9e99a64d74096b0282b23253ae58205a3e99f/mne_bids/tsv_handler.py#L30-L33

This is currently the only place where this leads to issues. I would patch this as proposed here and wait until we get issues elsewhere to do a bigger change (e.g., by handling `"n/a"` as np.nan within TSV handler and our codebase)